connectors/src/admin/cli.ts

```diff
@@ -74,6 +75,66 @@ const connectors = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+const notion = async (command: string) => {
+  switch (command) {
+    case "restart-all": {
+      const queue = new PQueue({ concurrency: 10 });
+      const promises: Promise<void>[] = [];
+      const connectors = await Connector.findAll({
```

Comment:
obviously this won't work forever @spolu

Reply:
:+1: